python/ray/_private/runtime_env.py

```diff
@@ -0,0 +1,298 @@
+import hashlib
+import logging
+import inspect
+
+from filelock import FileLock
+from pathlib import Path
+from zipfile import ZipFile
+from ray.job_config import JobConfig
+from enum import Enum
+from ray.experimental import internal_kv
+from ray.core.generated.common_pb2 import RuntimeEnv
+from typing import List, Tuple
+from types import ModuleType
+from urllib.parse import urlparse
+import os
+import sys
+
+PKG_DIR = "/tmp/ray/runtime_resources"
```

Comment:
should this be scoped to the current session dir?

Reply:
The package will be shared by multiple jobs and the lifetime of it is independent of session dir. GC in GCS is the place manage when to delete it.

Reply:
ah yeah the session is scoped to the lifetime of the ray node in the cluster, not scoped to a job

Reply:
Got it. I'll update it.